protocol/src/main/java/io/zeebe/protocol/record/intent/JobIntent.java

```diff
@@ -16,31 +16,25 @@
 package io.zeebe.protocol.record.intent;
 
 public enum JobIntent implements ProcessInstanceRelatedIntent {
-  /**
-   * @deprecated to be removed after engine refactoring TODO (#6202) remove at the end of engine
-   *     refactoring
-   */
-  @Deprecated
-  CREATE((short) 0),
-  CREATED((short) 1),
+  CREATED((short) 0),
```

Comment:
What is the reason to change the other enum constants' short values? I'm not certain that is beneficial here. What if in the future you'd want to bring it back?

Reply:
It will not come back :zombie: